src/main/java/com/stripe/param/PaymentIntentCreateParams.java

```diff
@@ -11,7 +11,13 @@
 import lombok.Getter;
 
 public class PaymentIntentCreateParams extends ApiRequestParams {
-  /** Amount intended to be collected by this PaymentIntent. */
+  /**
+   * A positive integer representing how much to charge in the [smallest currency
+   * unit](https://stripe.com/docs/currencies#zero-decimal) (e.g., 100 cents to charge $1.00 or 100
+   * to charge ¥100, a zero-decimal currency). The minimum amount is $0.50 US or [equivalent in
+   * charge
+   * currency](https://support.stripe.com/questions/what-is-the-minimum-amount-i-can-charge-with-stripe).
```

Comment:
do those markdown-like links work in javadoc?

Reply:
No unfortunately. We should do one clean up on this. But the link still works.

Reply:
We should look into using [this project](https://github.com/Abnaxos/markdown-doclet) to enable the use of Markdown syntax in Javadoc.